.swiftlint.yml

```diff
@@ -1,12 +1,76 @@
+#
+# Automatically downloaded from https://raw.githubusercontent.com/raywenderlich/swift-style-guide/master/com.raywenderlich.swiftlint.yml by SwiftLint on 22/08/2022 at 4:40:02 pm.
+#
+
+parent_config: https://raw.githubusercontent.com/raywenderlich/swift-style-guide/master/com.raywenderlich.swiftlint.yml
+
 included: # paths to include during linting. `--path` is ignored if present.
   - Source
-  - iOS Example
   - iOS Example Frame
   - iOS Example Frame Carthage
   - iOS Example Frame SPM
-excluded: # paths to ignore during linting. Takes precedence over `included`.
+  - Tests
+  - scripts
+  - Documentation
+
+excluded:
+  - ${PWD}/Carthage
+  - ${PWD}/Pods
+  - ${PWD}/DerivedData
+  - iOS Example Frame/Pods
   - Carthage
-  - Pods
-identifier_name:
-  excluded:
-    - id
+  - Checkout
+  - iOS Example Apple Pay
+  - iOS Example
+
+disabled_rules:
+  - discarded_notification_center_observer
+  - notification_center_detachment
+  - orphaned_doc_comment
+  - todo
+  - unused_capture_list
+  - switch_case_alignment
+  - multiline_arguments
+  - multiline_parameters_brackets
+  - multiline_literal_brackets
+  - class_delegate_protocol
+
+attributes:
+  always_on_same_line:
+    - "@IBSegueAction"
+    - "@IBAction"
+    - "@NSManaged"
+    - "@objc"
+    - "@frozen"
+
+indentation_width:
+  indentation_width: 4
```

Comment:
As discussed we use 2 spaces in checkout iOS SDK as per Ray Wanderlich rules. Its upto us if we want to sync this between Checkout iOS SDK and Frames to keep 2 spaces. Otherwise we will have 2 different set of rules.

Reply:
The 2 spaces should really go. I get that it was an interesting thing to try, but Xcode newly installed uses 4 spaces, same as most open sourced projects and all Apple examples. This 2 space is just trying to appear different than the standard without any good reason. I'd like if we try to standout via the amazing product we create, not funky code styling, so would say we should keep the 4 spaces going forward.

Reply:
Makes sense when we decided this it was more of a voting thing. I am happy to go back to 4 spaces. @ehab-al-cko I think what you can do is once this PR is merged and whatever rules we decide here You can create a ticket in backlog to sync similar style with Checkout iOS SDK to keep things in sync.

Reply:
Checkout is a target of the frames project now, so in all fairness we don't need to think of it as "something else". If you'll look at the files changed, there are a number of changes already being applied on files within the Checkout folder, which makes me think this is already applied.

Reply:
@deepesh-vasthimal-cko [[iOS] - Apply same Frames swiftlint configurations to Checkout SDK](https://checkout.atlassian.net/browse/PIMOB-1405?atlOrigin=eyJpIjoiMmFiOTk3OTUwYTUwNGIxNGFmNDdmYzAyM2ZjM2NmZWIiLCJwIjoiaiJ9)